test/e2e-tests.sh

```diff
@@ -44,14 +47,26 @@ function knative_setup() {
 }
 
 function knative_teardown() {
+  uninstall_istio
```

Comment:
Should this be at the end of `knative_teardown`?

Reply:
Yes that would be better, moved it.
BTW, `knative_teardown` and `test_teardown` will only be executed if this script is run locally, so it would not impact the CI flow.